src/components/Form/Label/index.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import classNames from 'classnames';
+import PropTypes from 'prop-types';
+import styles from './label.module.scss';
+
+const FormLabel = ({ label, required, error, vertical, as = 'a', children }) => {
```

Comment:
`error` is an object because this is how `react-hook-form` sends errors. If there is no error, `error` is undefined, so I think it is easier to pass an object instead of using an optional chaining operator like `<FormLabel error={errors.field?.message}>` every time this component is used.